cmake/FindHEIF.cmake

```diff
@@ -0,0 +1,14 @@
+# - Try to find HEIF
+# Once done, this will define
+#
+#  HEIF_FOUND - system has HEIF
+#  HEIF_INCLUDE_DIR - the HEIF include directories
+#  HEIF_LIBRARY - link this to use HEIF
+
+MESSAGE(STATUS "Try to find HEIF")
```

Comment:
I recommend you to take a look to this CMake functionality used when writing FindXXX.cmake modules

https://cmake.org/cmake/help/v3.0/module/FindPackageHandleStandardArgs.html

Right now you are assuming that the library is found. We would need to check if the DIR and LIBRARY has been found.